nix/mkDevShell.nix

```diff
@@ -12,21 +13,37 @@ let
 
   bky-as-stable = pkgs.stdenv.mkDerivation {
     pname = "bky-as";
-    version = version;
+    version = asVersion;
     src = builtins.fetchurl {
-      url = "https://github.com/blocky/attestation-service-cli/releases/download/${version}/bky-as_${goos}_${goarch}";
+      url = "https://github.com/blocky/attestation-service-cli/releases/download/${asVersion}/bky-as_${goos}_${goarch}";
     };
     unpackPhase = ":";
     installPhase = ''
       install -D -m 555 $src $out/bin/bky-as
     '';
   };
 
+  bky-c-stable = pkgs.stdenv.mkDerivation {
+    pname = "bky-c";
+    version = cVersion;
+    src = builtins.fetchurl {
+      # todo: switch bky-as to bky-c
+      # url = "https://github.com/blocky/compiler/releases/download/${cVersion}/bky-c_v${cVersion}_${goos}_${goarch}";
```

Comment:
Looking at this URI, I do wonder why we are adding the version to the executable name since it is further up the path....FWIW, it doesn't matter, I am curious as most other projects that use goreleaser I have looked at do something closer to `bky-c_${goos}_${goarch}`.

Reply:
looking at a few other projects... looks like it is split,